pyEpiabm/pyEpiabm/tests/test_routine/test_space_forces.py

```diff
@@ -0,0 +1,52 @@
+import unittest
+
+import pyEpiabm as pe
+from pyEpiabm.property.infection_status import InfectionStatus
+from pyEpiabm.routine import SpatialInfection
+
+
+class TestSpatialInfection(unittest.TestCase):
+    """Test the 'PlaceInfection' class, which contains the
+    infectiousness and susceptibility calculations that
+    determine whether infection events occur within places.
+    Each function should return a number greater than 0.
+    """
+    @classmethod
+    def setUpClass(cls) -> None:
+        """Intialise a population with one infector and one
+        infectee, both in the same place and household.
+        """
+        cls.cell = pe.Cell()
+        cls.microcell = pe.Microcell(cls.cell)
+        cls.infector = pe.Person(cls.microcell)
+        cls.infectee = pe.Person(cls.microcell)
+        cls.timestep = 1
+
+    def test_calc_place_susc(self):
```

Comment:
```suggestion
    def test_calc_space_susc(self):
```

I'm not saying this was a copy and paste job but you should maybe change the test names! If you wouldn't mind doing me a favour and removing '_calc' from each test name in this file and `test_place_forces.py` that would be much appreciated!

Reply:
oops so att-ed